bus-mapping/src/rpc.rs

```diff
@@ -119,6 +119,17 @@ impl<P: JsonRpcClient> GethClient<P> {
             .map_err(|e| Error::JSONRpcError(e.into()))?;
         Ok(resp.0.into_iter().map(|step| step.result).collect())
     }
+
+    /// Calls `eth_getCode` via JSON-RPC returning a contract code
+    pub async fn get_code_by_address(
+        &self,
+        contract_address: String,
```

Comment:
nitpick: For contract address, use `Address` instead of `String` would be better, what do you think?

Reply:
Thank you for the review!
I think that's more beautiful.
I am going to change.